protobuf/source_reflection.go

```diff
@@ -111,5 +118,10 @@ func (s *grpcreflectSource) FindService(fullyQualifiedName string) (*desc.Servic
 }
 
 func (s *grpcreflectSource) Close() {
+	s.ctxCancel()
 	s.client.Reset()
 }
+
+func wrapReflectionError(err error) error {
+	return fmt.Errorf("error in protobuf reflection: %v", err)
```

Comment:
nit: you can use `fmt.Errorf("error in protobuf reflection: %s", err.Error())` (faster, less alloc)

Reply:
For consistency in the codebase, going to stick to `%v` (this method won't impact benchmark or call performance)